Documentation/Upgrade/rook-upgrade.md

```diff
@@ -51,6 +52,12 @@ those releases.
 
 ## Breaking changes in v1.16
 
+* The minimum supported Kubernetes version is v1.27.
+
+* Rook no longer supports Ceph v17 (Quincy) as it is end of life. Rook v1.15 clusters should
+    [upgrade Ceph](./ceph-upgrade.md) to at least Ceph v18 (Reef) before upgrading Rook to v1.16.
+    Be aware of known [Ceph v18.2.4 ARM issues](https://github.com/rook/rook/issues/14502).
+
 * Rook fully deprecated CSI "holder" pods in Rook v1.16. Any Rook v1.15 CephCluster with
     `csi-*plugin-holder-*` pods present in the Rook operator namespace must follow holder pod
     removal migration steps outlined in
```

Comment:
Depending on whether the ObjectStore changes @arttor is working on are merged before release, we might want to add a deprecation note at the end about changes to the Swift API.